ACPs/118-warp-signature-request/README.md

```diff
@@ -0,0 +1,90 @@
+
+## Motivation
+
+Warp message signatures consist of an aggregate BLS signature composed of the individual signatures of a subnet's validators. Individual signatures need to be retreivable by the party that wishes to construct an aggregate signature. At present, this is left to VMs to implement, as is the case with [Subnet EVM](https://github.com/ava-labs/subnet-evm/blob/v0.6.6/plugin/evm/message/signature_request.go#20) and [Coreth](https://github.com/ava-labs/coreth/blob/v0.13.6-rc.0/plugin/evm/message/signature_request.go#L20)
+
+This creates friction in applications that are intended to operate across many VMs (or distinct implementations of the same VM). As an example, the reference Warp message relayer implementation, [awm-relayer](https://github.com/ava-labs/awm-relayer), fetches individual signatures from validators and aggregates them before sending the Warp message to its destination chain for verification. However, Subnet EVM and Coreth have distinct codecs, requiring the relayer to [switch](https://github.com/ava-labs/awm-relayer/blob/v1.4.0-rc.0/relayer/application_relayer.go#L372) according to the target codebase.
+
+Another example is ACP-75, which aims to implement acceptance proofs using Warp. The signature aggregation mechanism is not [specified](https://github.com/avalanche-foundation/ACPs/blob/main/ACPs/75-acceptance-proofs/README.md#signature-aggregation), which is a blocker for that ACP to be marked implementable.
+
+Standardizing the Warp Signature Request format by implementing it as an AppRequest type in AvlanahceGo would simplify the implementation of ACP-75, and streamline signature aggregation for out-of-protocol services such as Warp message relayers.
+
+## Background
+Subnet EVM and Coreth currently handle Warp message signature requests from peers by decoding the `AppRequest` bytes (forwarded from Avalanchego via the `AppHandler` interface) using a types registered with their message codec:
+
+```
+type MessageSignatureRequest struct {
+	MessageID ids.ID
+}
+```
+and
+```
+type BlockSignatureRequest struct {
+	BlockID ids.ID
+}
+```
+
+Signatures are provided via the following `AppResponse` message type:
+
+```
+type SignatureResponse struct {
+	Signature [bls.SignatureLen]byte
+}
+```
+
+`MessageID` represents the VM-agnostic Warp message ID, while `BlockID` represents a block hash that nodes are willing to attest to via Warp signature. and could be ported as-is to AvalancheGo. `BlockID` can instead be generalized to represent a 32-byte hash over some data. Interpreting that hash is left to VMs. For example, `Hash` could represent a transaction hash that a node only signs on receiving a `HashSignatureRequest`. 
+
+## Specificaiton
+
+We take the types currently defined in Subnet EVM and Coreth, modify them to not be EVM specific, and implement them as `AppRequest` message types will be defined as Protobuf specs. Corresponding `AppRequest` handlers will need to be implemented.
```

Comment:
This first sentence seems like a run-on, could you reword?

Also I think it should clarify that AppRequest handlers are left for the VMs to implement in order to adhere to this interface.